internal/pkg/exec/docker.go

```diff
@@ -159,3 +161,38 @@ func imageName(uri, tag string) string {
 	}
 	return fmt.Sprintf("%s:%s", uri, tag)
 }
+
+// GetHelperProviderFromDockerCfg will fetch credential store used by docker from config file
+func (c DockerCommand) GetHelperProviderFromDockerCfg() (string) {
+	// Look into the default locations
+	pathsToTry := []string{"/.docker/config.json", ".dockercfg"}
+	for _, path := range pathsToTry {
+		content, err := ioutil.ReadFile(fmt.Sprintf("%s/%s",os.Getenv("HOME"), path))
+		if err != nil {
+			// if we can't read the file keep going
+			continue
+		}
+
+		provider, err := parseCredFromDockerConfig(content)
+		if err != nil {
+			continue
+		}
+		if provider != "" {
+			return provider
+		}
+	}
+	return ""
+}
+
+// parseCredFromDockerConfig will unmarshal string to struct
```

Comment:
Maybe we don't need this comment for private function since it doesn't add a lot of value.

Reply:
Removed